test/Microsoft.AspNetCore.Mvc.FunctionalTests/InputObjectValidationTests.cs

```diff
@@ -243,5 +245,22 @@ public async Task ValidationProviderAttribute_RequiredAttributeErrorMessage_Will
             var responseContent = await response.Content.ReadAsStringAsync();
             Assert.Equal(expectedError, actual: responseContent);
         }
+
+        // Test for https://github.com/aspnet/Mvc/issues/7357
+        [Fact]
+        public async Task ValidationThrowsError_WhenValidationExceedsMaxValidationDepth()
+        {
+            // Arrange
+            var expected = $"ValidationVisitor exceeded the maximum configured validation depth '32' when validating property 'Value' on type '{typeof(RecurisiveIdentifier)}'. " +
```

Comment:
Typo (a new typo in same identifier, this time an extra "i"): `RecurisiveIdentifier` -> `RecursiveIdentifier`